### PR TITLE
Update RNA-Seq-hands-on.md

### PR DESCRIPTION
**Pull request by pavanvidem**
All tool name occurrences standardized

### CODE REVIEW

File: RNA-Seq/RNA-Seq-hands-on.md [hunk 1]
```diff
@@ -40,32 +40,32 @@ This exercise uses RNA-seq data from the study by [Brooks et al. 2011](http://ge
 - Re-run **FastQC** and inspect the differences.
 
 
-**Step 2: Mapping of the reads with TopHat**
+**Step 2: Mapping of the reads with TopHat2**
```
Comment (bgruening) [inline RNA-Seq/RNA-Seq-hands-on.md]: I think in galaxy the tool is actually called tophat.

Comment (pavanvidem) [inline RNA-Seq/RNA-Seq-hands-on.md]: let's change all of them to TopHat then. In previous version, it was called TopHat2 sometimes.

Comment (bgruening) [inline RNA-Seq/RNA-Seq-hands-on.md]: :+1:

**Comment by bgruening** [comment]
@pavanvidem can we close this?

**Comment by pavanvidem** [comment]
yes please.

**Pull request closed without merging**